src/utils/layer-utils/layer-utils.js

```diff
@@ -100,20 +100,38 @@ export function getLightSettingsFromBounds(bounds) {
     : DEFAULT_LIGHT_SETTINGS;
 }
 
-export function getTimeAnimationDomainFoTripLayer(layer, datasets) {
+export function getTimeAnimationDomainForTripLayer(layer, datasets) {
   const {columns, dataId} = layer.config;
   const dataContent = datasets[dataId].allData;
   const geojsonFieldIdx = columns.geojson.fieldIdx;
 
-  // TODO: H factor in more input format beyond 3rd coord as ts
-  const timeField = dataContent
-    .map(d => d[geojsonFieldIdx].geometry.coordinates.map(coord => coord[3]))
-    .flat();
+  // TODO: factor in more input format beyond 3rd coord as ts
 
-  return extent(timeField);
+  const minTs = [];
+  const maxTs = [];
+  dataContent.map(d => {
```

Comment:
Don't call `extent` 3 times. it's expensive
You can adapt the `reduce` function I wrote in `updateAnimationDomain` here